### DISCUSSION TITLE
ripes

### DISCUSSION THREAD

**Discussion by alirafee21** [mortbopet/Ripes#194] [Q&A] [answered]
.data

.globl main
.text

main:
LOOPINIT: # Many loops have an initialization section.
li $t0, 0
WHILE: # The loop checks the condition, then evaluates the body.
li t1, 5
bge t0, t1, DONE
addi t0, t0, 1
j WHILE
DONE:

why do I get an unknown register $t0 error?

**Comment by mortbopet** [comment, marked as answer]
As far as i am aware of, prefixing registers with $ is not part of the spec for the RISC-V assembly language (which is why ou're getting an error - this is not supported in Ripes).
